appengine/flexible/static_files_spa/app.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: python
+env: flex
```

Comment:
Why not standard instead of Flex? It is less complicated for the developer, and has all the capabilities used in the sample.

Reply:
Absolutely. But I placed the new sample near an existing one - static_files, plus standard samples folder doesn't have appengine samples. But in my sample there's additional GAE config - app.standard.yaml for standard env.